content/blog/2020/2020-10-01-document-embedders.md

```diff
@@ -0,0 +1,73 @@
++++
+author = "Primož Godec and Nikola Djukič"
+date = "2020-10-01"
+draft = false
+title = "How to identify fake news by document embeddings"
```

Comment:
I would say **with** document embeddings rather than **by**. I think by is used more as passive or with gerund (https://www.grammar-quizzes.com/gerund3.html). No expert here, though.